migrate/20220309224100_create_dao_coin_limit_order_tables.go

```diff
@@ -41,21 +41,14 @@ func init() {
 		// Create pg_dao_coin_limit_orders table.
 		_, err = db.Exec(`
 			CREATE TABLE pg_dao_coin_limit_orders (
+				order_id                                           BYTEA PRIMARY KEY,
 				transactor_pkid                                    BYTEA NOT NULL,
 				buying_dao_coin_creator_pkid                       BYTEA NOT NULL,
 				selling_dao_coin_creator_pkid                      BYTEA NOT NULL,
 				scaled_exchange_rate_coins_to_sell_per_coin_to_buy TEXT NOT NULL,
 				quantity_to_fill_in_base_units                     TEXT NOT NULL,
 				operation_type                                     BIGINT NOT NULL,
-				block_height                                       BIGINT NOT NULL,
-
-				PRIMARY KEY (
-					transactor_pkid,
-					buying_dao_coin_creator_pkid,
-					selling_dao_coin_creator_pkid,
-					scaled_exchange_rate_coins_to_sell_per_coin_to_buy,
-					block_height
-				)
+				block_height                                       BIGINT NOT NULL
```

Comment:
It may be useful for us to include some additional indices here:
* `transactor_pkid`: This would help power the GetOrdersByTransactorPKID query.
* `buying_dao_coin_creator_pkid, selling_dao_coin_creator_pkid`: This would help power the GetMatchingOrders query.
* `order_id`: This is the primary key so we already have an index here. This helps power the GetOrdersByOrderID query. :heavy_check_mark: 
@lazynina LMK what you think.

Reply:
Added w/ https://github.com/deso-protocol/core/pull/327/commits/f6ede7c1cc02a3139c289f4abfbdcab4e2aa7d0e.